controllers/suite_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	backupv1beta1 "github.com/open-cluster-management-io/cluster-backup-operator/api/v1beta1"
+	restorev1beta1 "github.com/open-cluster-management-io/cluster-backup-operator/api/v1beta1"
```

Comment:
Reason for this?

Reply:
trying to test if adding restorev1beta1 into the scheme works with no error

Reply:
yeah but the test should be functional: my understanding is that testing the idempotency of scheme adding (even with different import alias) should not stay in our operator.